src/app/app.module.ts

```diff
@@ -4,14 +4,17 @@ import { BrowserModule } from '@angular/platform-browser';
 import { AppRoutingModule } from './app-routing.module';
 
 import { AppComponent } from './app.component';
+import { ButtonComponent } from './shared/components/button/button.component';
 import { HeaderComponent } from './components/header/header.component';
-import { LogoComponent } from './components/logo/logo.component';
+import { LogoComponent } from './components/header/logo/logo.component';
 import { FooterComponent } from './components/footer/footer.component';
-import { CoursesComponent } from './components/courses/courses.component';
-import { BreadcrumbsComponent } from './components/breadcrumbs/breadcrumbs.component';
+import { CoursesComponent } from './components/courses-page/courses/courses.component';
+import { BreadcrumbsComponent } from './components/courses-page/breadcrumbs/breadcrumbs.component';
 import { CoursesPageComponent } from './components/courses-page/courses-page.component';
 import { LoginComponent } from './components/login/login.component';
-import { SectionComponent } from './components/section/section.component';
+import { SearchComponent } from './components/courses-page/search/search.component';
+import { CoursesItemComponent } from './components/courses-page/courses/courses-item/courses-item.component';
+import { FormsModule } from '@angular/forms';
```

Comment:
I recommend applying the following order for import:

1. Core Angular imports(angular/core, angular/common, angular/forms etc.)
Third-Party Library Imports(angular-material, rxjs etc)
Application Imports(SharedModule, FeatureModule etc)
2. Interfaces, enums, outer constants
3.Service imports
4.Component imports
In such way it will be easier to other developers to examine the imports :)

Reply:
ok, got it